harper-core/Cargo.toml

```diff
@@ -21,9 +22,10 @@ smallvec = { version = "1.13.2", features = ["serde"] }
 thiserror = "1.0.65"
 unicode-blocks = "0.1.9"
 unicode-width = "0.2.0"
+levenshtein_automata = { version = "0.2.1", features = ["fst_automaton"] }
 
 [dev-dependencies]
-criterion = "0.5.1"
+criterion = { version = "0.5.1", features = ["plotters", "cargo_bench_support"], default-features = false}
```

Comment:
Unless this directly effects our ability to run the benchmarks, let's make these changes in a separate PR.